app/views/web_git/commands/status.html.erb

```diff
@@ -44,9 +44,8 @@
                   <%= form_tag commits_url do %>
                     <div class="form-group">
                       <%= label_tag :title %>
-                      <%= text_field_tag :title, nil, placeholder: "Enter a title for your commit (required)...", class: "form-control", required: true, autofocus: true %>
+                      <%= text_field_tag :title, nil, placeholder: "Enter a title for your commit (required)...", class: "form-control", required: true, autofocus: true, html: { autocomplete: "off" } %>
```

Comment:
I don't think this is rendering the right html. It creates this:
![patrick-off-autocomplete](https://user-images.githubusercontent.com/17581658/62233707-b8488880-b38e-11e9-81ab-35c91a9fb355.PNG)

The html should render kind of [like this](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#Disabling_autocompletion) from what I understand. Although, there appears to be [some debate](https://stackoverflow.com/a/51686510) over how browsers choose to follow this setting.

According to [this answer](https://stackoverflow.com/a/24781882), it looks like that's how you disable autocomplete on the `form_tag` method but not a `text_field_tag`.